openshift/templates/images/wordpress/docker/Dockerfile

```diff
@@ -1,7 +1,18 @@
 FROM wordpress:6.0.0-php7.4-fpm-alpine
 # Install LDAP for Next Active Directory Integration Plugin.
 RUN set -ex; \
-	apk add --no-cache --virtual .build-deps \
+        apk add --no-cache --virtual .build-deps \
         openldap-dev; \
-	docker-php-ext-install ldap; \
-    docker-php-ext-enable ldap; \
\ No newline at end of file
+        docker-php-ext-install ldap; \
+        docker-php-ext-enable ldap;
+
+RUN mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini; \
```

Comment:
just curious why you choose this method, as opposed to having a file and copying it?
I worry that something in translation might not translate completely 
The file method also seems more simplistic..

Reply:
As noted in the PR, wasn't certain of best practice and used examples I had available at the time, hoping for suggestions like this one to improve it. Thanks.

Looking at the wordpress variables more closely, it looks like it might change again anyway.